libstuff/STCPManager.cpp

```diff
@@ -339,3 +332,22 @@ bool STCPManager::Socket::recv() {
     }
     return result;
 }
+
+unique_ptr<STCPManager::Port> STCPManager::openPort(const string& host) {
```

Comment:
This may turn into a `Port` constructor in the future, but this is as far as I wanted to push this refactor in this PR.